### PR TITLE
Help docs: edit fly m run help text

### PR DESCRIPTION
**Pull request by andie787**
### Change Summary

What and Why:
Clarify some help doc text.

How:
Edits files.

Related to:

---

### Documentation

- [ ] Fresh Produce
- [ ] In superfly/docs, or asked for help from docs team
- [x] n/a

### CODE REVIEW

**Review by catflydotio — APPROVED** [review]
A definite improvement! I haven't built flyctl; my comments and approval are based on text in the PR.

File: internal/command/machine/run.go [hunk 1]
```diff
@@ -152,20 +152,20 @@ var runOrCreateFlags = flag.Set{
...
 	flag.Bool{
 		Name:        "rm",
-		Description: "Automatically remove the machine when it exits",
+		Description: "Automatically remove the Machine when it exits",
 	},
 	flag.StringSlice{
 		Name:        "volume",
 		Shorthand:   "v",
-		Description: "Volumes to mount in the form of <volume_id_or_name>:/path/inside/machine[:<options>]",
+		Description: "Volumes to mount, in the form of <volume_id_or_name>:/path/inside/machine[:<options>]",
```
[the remark this answers is not in the document]
Comment (andie787) [reply internal/command/machine/run.go:168]: Looking into this!

[the remark this answers is not in the document]
Comment (andie787) [reply internal/command/machine/run.go]: will follow up next PR

File: internal/command/machine/run.go [hunk 2]
```diff
@@ -228,19 +228,19 @@ func newRun() *cobra.Command {
...
 			Name:        "user",
-			Description: "Username, if we're shelling into the machine now.",
+			Description: "Used with --shell. The username, if we're shelling into the machine now.",
 			Default:     "root",
 			Hidden:      false,
 		},
 		flag.String{
 			Name:        "command",
-			Description: "Command to run, if we're shelling into the machine now (in case you don't have bash).",
+			Description: "Used with --shell. The command to run, if we're shelling into the machine now (in case you don't have bash).",
```
[the remark this answers is not in the document]
Comment (andie787) [reply internal/command/machine/run.go]: to follow up

**Pull request merged**